docs/en/latest/plugins/limit-count.md

```diff
@@ -46,7 +46,10 @@ The `limit-count` Plugin limits the number of requests to your service by a give
+| redis_username          | string  | False                                     |               |                                        | Username of the Redis server. Used when the `policy` is set to `redis`.                                                                                                                                                                                                                                                                                                                                                                                                           |
 | redis_password          | string  | False                                     |               |                                        | Password of the Redis server. Used when the `policy` is set to `redis` or `redis-cluster`.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| redis_ssl               | boolean | False                                     | false         |                                        | If set to `true`, then uses SSL to connect to redis instance. Used when the `policy` attribute is set to `redis`.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| redis_ssl_verify        | boolean | False                                     | false         |                                        | If set to `true`, then verifies the validity of the server SSL certificate. Used when the `policy` attribute is set to `redis`.                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Could you paste a link point to the https://github.com/openresty/lua-nginx-module#tcpsocksslhandshake for the  `redis_ssl_verify` variable?